app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewHolderFactory.kt

```diff
@@ -207,6 +207,7 @@ class ChangeMastercopyViewHolder(private val viewBinding: ItemTxChangeMastercopy
             finalStatus.alpha = OPACITY_FULL
             dateTime.alpha = viewTransfer.alpha
             version.alpha = viewTransfer.alpha
+            label.alpha = viewTransfer.alpha
```

Comment:
Shouldn't nonce and moduleAddress also have the alpha set? Maybe reorder, so that we have three blocks: texts, alphas and visibility.

Reply:
IMHO txTypeIcon should also be greyed out: 
`txTypeIcon.alpha = viewTransfer.alpha`

Reply:
**Not the chevron!** Sorry for the confusion. 
But the txTyoeIcon needs to be alpha'ed too.

Reply:
And there is no example wit a nonce in the design. So I will ask Sasha.